### DISCUSSION TITLE
Dashboard Company Overview for Multi-Client Organisations

### DISCUSSION THREAD

**Discussion by noahammond** [snipe/snipe-it#13403] [Ideas]
Hi,

I've been using Snipe-IT in my organisation for about 2 months now and I'm loving it. However, I feel that it could do with some extra features, my main one being a way to select and view 1 company's assets instead of all assets.

I've managed to create a workaround for this by creating a separate admin account that only has permissions to view a certain company but logging in and out is too tedious (already have enough of that with 365)

A feature on the dashboard to view each company's asset status would be good or a feature at the top of the page to select which company's assets you're currently viewing.

Just a little suggestion I had, hopefully it can be implemented.
Thanks,

Noah

**Comment by snipe** [comment]
You should be able to see all of a company's assets by drilling down into the company itself.

https://demo.snipeitapp.com/companies/1

<img width="1587" alt="Screenshot 2023-08-03 at 2 11 13 PM" src="https://github.com/snipe/snipe-it/assets/197404/26681606-6216-4451-9978-b38f00bf0684">

**Reply by noahammond** [reply]
Yes I'm aware of this, my problem is getting to this page isn't as simple as clicking All Assets or Dashboard. It would be nice to have an optional button to view all companies, etc?

Ideally it would be great if we could customize the dashboard's widgets. For example: a customizable widget that can report the amount of assets per company, or available assets per company etc.

Edit: Just spoke to some of my colleagues and they suggested a filter similar to Excel's table filter. We've got a sort feature but not a filter feature.

**Comment by snipe** [comment]
> Ideally it would be great if we could customize the dashboard's widgets. For example: a customizable widget that can report the amount of assets per company, or available assets per company etc.


This is already on the roadmap.

**Reply by noahammond** [reply]
Ah brilliant! Is there anywhere we can view the roadmap or is just something internal?
